### PR TITLE
Fix fix tree

### PR DESCRIPTION
**Pull request by mkoistinen**
Fixes (? please review!) #4456.

### CODE REVIEW

**Comment by mkoistinen** [comment]
@yakky, @czpython please review!

**Comment by yakky** [comment]
@mkoistinen LGTM

File: cms/models/pluginmodel.py [hunk 1]
```diff
@@ -334,21 +334,26 @@ def copy_plugin(self, target_placeholder, target_language, parent_cache, no_sign
...
-                for plugin in CMSPlugin.objects.filter(placeholder_id=placeholder.pk,
-                                                       language=language).order_by('depth', 'path'):
-                    order = CMSPlugin.objects.filter(parent_id=plugin.pk).values_list('pk', flat=True)
+
+                for plugin in CMSPlugin.objects.filter(
+                        placeholder_id=placeholder.pk,
+                        language=language).order_by('depth', 'path'):
+                    order = CMSPlugin.objects.filter(
+                            parent_id=plugin.pk
+                        ).order_by('position').values_list('pk', flat=True)
```
Comment (czpython) [inline cms/models/pluginmodel.py]: indentation is a bit odd, I would match closing parenthesis with the variable declaration, nothing major though.

File: cms/models/pluginmodel.py [hunk 2]
```diff
@@ -334,21 +334,26 @@ def copy_plugin(self, target_placeholder, target_language, parent_cache, no_sign
...
         super(CMSPlugin, cls).fix_tree(destructive)
         for placeholder in Placeholder.objects.all():
             for language, __ in settings.LANGUAGES:
-                order = CMSPlugin.objects.filter(placeholder_id=placeholder.pk, language=language,
-                                                 parent_id__isnull=True
-                                                 ).values_list('pk', flat=True)
+                order = CMSPlugin.objects.filter(
+                        placeholder_id=placeholder.pk, language=language,
+                        parent_id__isnull=True
+                    ).order_by('position').values_list('pk', flat=True)
```
Comment (czpython) [inline cms/models/pluginmodel.py]: indentation is a bit odd, I would match closing parenthesis with the variable declaration, nothing major though.

**Comment by czpython** [comment]
:+1:

**Pull request merged**